### PR TITLE
Descriptions for tx, nodeStatus and peerChainState

### LINKED ISSUE #312: Document GraphQL API (node state)

### PR DESCRIPTION
**Pull request by longfin**
This PR adds descriptions for tx, nodeStatus and peerChainState queries.

it resolves #312 and #313

### CODE REVIEW

File: NineChronicles.Headless/GraphTypes/NodeStatus.cs [hunk 1]
```diff
@@ -29,11 +29,17 @@ public class NodeStatusType : ObjectGraphType<NodeStatusType>
...
                 resolve: context => context.Source.BootstrapEnded);
-            Field<NonNullGraphType<BooleanGraphType>>(name: "preloadEnded",
+            Field<NonNullGraphType<BooleanGraphType>>(
+                name: "preloadEnded",
+                description: "Whether the current libplanet node had ended preloading.",
                 resolve: context => context.Source.PreloadEnded);
-            Field<NonNullGraphType<BlockHeaderType>>(name: "tip",
+            Field<NonNullGraphType<BlockHeaderType>>(
+                name: "tip",
+                description: "The block header of tip block from the current canonical chain.",
```
Comment (area363) [inline NineChronicles.Headless/GraphTypes/NodeStatus.cs:42]:
```suggestion
                description: "Block header of the tip block from the current canonical chain.",
```

**Review by area363 — COMMENTED** [review]
🙏

**Review by moreal — APPROVED** [review]

**Review by riemannulus — APPROVED** [review]

**Review by area363 — APPROVED** [review]

**Comment by ipdae** [comment]
we need to update schema.

**Comment by moreal** [comment]
> 
> 
> we need to update schema.

The schema build has been automated since #382. @longfin It seems okay only to rebase

File: NineChronicles.Headless/GraphTypes/TransactionType.cs [hunk 2]
```diff
@@ -11,34 +11,42 @@ public TransactionType()
...
                 resolve: context => context.Source.Nonce
             );
             Field<NonNullGraphType<PublicKeyType>>(
                 nameof(Transaction<T>.PublicKey),
+                description: "A PublicKey of the account who signed this transaction.",
                 resolve: context => context.Source.PublicKey
             );
             Field<NonNullGraphType<ByteStringType>>(
                 nameof(Transaction<T>.Signature),
+                description: "A digital signature of the content of this Transaction<T>.",
```
Comment (moreal) [inline NineChronicles.Headless/GraphTypes/TransactionType.cs:29]: Is it enough to `transaction` instead of `Transaction<T>` like other descriptions do?

**Review by ipdae — APPROVED** [review]

**Review by area363 — APPROVED** [review]

**Review by moreal — APPROVED** [review]

**Pull request merged**